server/gateway/api/swagger.yaml

```diff
@@ -479,9 +489,16 @@ definitions:
         type: "string"
         enum: ["active", "inactive"]
         description: "The firstname of the user"
+      createdAt:
```

Comment:
Does these have to be defined? 
If they are on the model they will be returned,
Are we modifying these in any ways?

Reply:
the swagger definitions are what is used to create the api documentation. If you omit properties then whoever client that consumes your api will not have the correct response structure. So you need to ensure the swagger documentation accurately reflect the actual response that is being returned

Reply:
👍🏼